packages/ckeditor5-engine/tests/view/placeholder.js

```diff
@@ -632,12 +633,62 @@ describe( 'placeholder', () => {
 			enablePlaceholder( {
 				view,
 				element: viewRoot,
-				text: 'foo bar baz',
 				isDirectHost: false
 			} );
 			viewRoot.placeholder = 'new placeholder';
 
 			expect( viewRoot.getChild( 0 ).getAttribute( 'data-placeholder' ) ).to.equal( 'new placeholder' );
 		} );
+
+		it( 'should through warning once if "text" is used as argument', () => {
```

Comment:
What did you mean by "should through warning once"?

Reply:
Warning should be throughn only once in dev console